packages/renderless/src/tabs/index.ts

```
@@ -131,15 +131,20 @@ export const calcMorePanes =
 
     if (tabs && tabs.length) {
       let tabsAllWidth = 0
+
+      if (state.currentIndex === -1) {
+        state.currentIndex = state.panes.findIndex((item) => item.state.paneName === state.currentName)
+      }
+      const currentIndex = state.currentIndex < 0 ? 0 : state.currentIndex
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)